packages/edit-site/src/components/global-styles/dimensions-panel.js

```diff
@@ -48,17 +49,20 @@ export default function DimensionsPanel() {
 	const styleWithLayout = useMemo( () => {
 		return {
 			...style,
-			layout: settings.layout,
+			layout: userSettings.layout,
 		};
-	}, [ style, settings.layout ] );
+	}, [ style, userSettings.layout ] );
 
 	const onChange = ( newStyle ) => {
 		const updatedStyle = { ...newStyle };
 		delete updatedStyle.layout;
 		setStyle( updatedStyle );
 
 		if ( newStyle.layout !== settings.layout ) {
```

Comment:
Can we change `settings.layout` here to `userSettings.layout`? The same is happening here as with the check in the blocks screen. If we compare against `userSettings` then at least this code won't run if both are `undefined` 😄  but it's probably best to do a deep equality comparison too.

Reply:
No, it's not necessary to do a deep equality for performance reasons IMO but yeah, we should compare to useSettings indeed.